app/templates/bestuurseenheid/zitting/agenda/index.hbs

```diff
@@ -57,18 +45,17 @@
 
       <div class="grid grid--is-stacked">
         {{#if model.agendapunten.length}}
-          <ctx.each @prop="sortedAgendapunten" as |elements agendapunt agendapuntCtx index|>
-            <div {{rdfa elements}} class="col--12-12">
-              <AgendapuntOverviewItem @agendapunt={{agendapunt}} @ctx={{agendapuntCtx}} @index={{index}} />
+          {{#each model.agendapunten as |agendapunt index|}}
+            <div resource={{agendapunt.uri}} typeof="besluit:Agendapunt" property="besluit:behandelt" class="col--12-12">
+              <AgendapuntOverviewItem @agendapunt={{agendapunt}} @index={{index}} />
```

Comment:
When I test this the component doesn't render anything here, could you double check ? :thinking:

Reply:
Now it's here, don't know what happened yesterday :p